package.json

```diff
@@ -86,6 +86,7 @@
     "babel-eslint": "^10.0.2",
     "babel-loader": "^8.0.0-beta.6",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.13",
+    "chromatic": "^5.10.0-canary.0",
```

Comment:
Any specific reason why we're using a canary build instead of a stable build?

Reply:
No, didn't even notice it. I just did `yarn add --dev chromatic`. Wouldn't expect it to install a non-stable build then 🤔

Reply:
Huh, maybe they configured it as a default version? Though judging from npm it doesn't look like it

<img width="782" alt="Screenshot 2021-08-13 at 19 23 50" src="https://user-images.githubusercontent.com/10011712/129396722-f99fe327-c7c8-4a5e-83f2-d41611a9cd79.png">

Reply:
Changed it to use the latest stable version instead.